src/removeItem.js

```diff
@@ -0,0 +1,9 @@
+export default function removeItem() {
```

Comment:
- [ ] It uses ES6 syntax

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png)


Kindly replace this function with the arrow function syntax.